lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -1107,13 +1107,22 @@ private BulkOperationResults<String, TaskanaException> completeTasks(
                 addErrorToBulkLog(this::checkPreconditionsForCompleteTask, bulkLog));
       } else {
         String userId = taskanaEngine.getEngine().getCurrentUserContext().getUserid();
+        User user = userMapper.findById(userId);
```

Comment:
should we maybe check  if additional User Data is configured?

taskanaEngine.getEngine().getConfiguration().getAddAdditionalUserInfo -> if this is false, then we don't need to make a database call at all, right?

Reply:
Yes, I will adjust it